bionic/persistence.py

```diff
@@ -754,43 +750,35 @@ class LocalFilesystem:
     to local disk.
     """
 
-    def __init__(self, inventory_dir, tmp_dir):
+    def __init__(self, inventory_dir):
         self.root_url = url_from_path(inventory_dir)
-        self.tmp_root_path = tmp_dir
+        self._fs = fsspec.filesystem("file")
```

Comment:
I am assuming that fsspec for "file" filesystem is able to work with urls?

Reply:
Yes, `file` is the protocol for the `LocalFileSystem` that we use here.